.github/workflows/dart-desktop-tests.yml

```diff
@@ -56,18 +61,20 @@ jobs:
           melos setup
 
       - name: Bump ulimit on macos
-        run: ulimit -n 10240
-        if: ${{ contains(inputs.os, 'macos') }}
+        run: |
+          ulimit -n
```

Comment:
Why do we do this?

Reply:
We don't, but I find it useful to see the current limit in the output of the step before setting it.